### PR TITLE
added the start of the login component

### PR DESCRIPTION
**Pull request by JakeGreer**

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import Vue from "vue";
 import Router from "vue-router";
 import Home from "pages/Home.vue";
```
Comment (theRealScoobaSteve) [inline src/router.js:3]: Can we remove Home and maybe the Landing folder?

**Review by theRealScoobaSteve — COMMENTED** [review]
Everything looks good, I don't know why the travis CI build didn't pass, looking into it now

File: src/router.js (hunk 1 above)
Comment (JakeGreer) [reply src/router.js:3]: Ya I will not sure why the home is in there. Do you not want folders inside the pages folder just files?

Comment (theRealScoobaSteve) [reply src/router.js:3]: Just files, .vue files can have the style, js, and template all in one file so should probably just do that, the styles folder is just for generic selectors we will use for our personal "bootstrap" css

**Review by theRealScoobaSteve — APPROVED** [review]

**Pull request merged**